assets/sass/_variables.scss

```diff
@@ -24,7 +24,7 @@ $secondary-bg-color: #f9f9f9;
 // Text Color
 $header-text-color: #5e5e5e;
 $dark-text-color: #5e5e5e;
-$medium-text-color: #777;
+$medium-text-color: #767676;
```

Comment:
`#777:#fff` → 4.47:1 FAIL
`#767676:#fff` → 4.54:1 PASS

Reply:
I might have a preference for aligning with the current Odyssey colors for subtle and body text, instead of introducing more arbitrary shades. But not a big deal, probably no practical difference.

Subtle text: `#6E6E78`
Body text: `#1D1D21`